### DISCUSSION TITLE
Newbie Developer Contributions

### DISCUSSION THREAD

**Discussion by jaydoiron** [mealie-recipes/mealie#1510] [General]
Hi all:
I'm new to Mealie, I'm a computer engineer by degree and a product manager by trade...looking for a fast-track approach to become a contributing developer to Mealie....but I don't know where to begin.  Any thoughts or suggestions?  I'm very interested in learning Vue.js and helping out on new front-end features.

**Comment by michael-genson** [comment]
Hey jaydoiron, welcome! First and foremost I highly recommend joining our discord, we've got a few active developers there.
https://discord.gg/sKfa8JUzx7

Second, there's been a ton of work in implementing developer resources, which makes it pretty painless to set up a new dev instance (at least compared to building it all manually!):

Everything goes through GitHub:
https://nightly.mealie.io/contributors/developers-guide/code-contributions/

And you can develop on a pre-built dev container in VSCode using docker:
https://nightly.mealie.io/contributors/developers-guide/starting-dev-server/

Lastly, check out the development channel on discord, there might be some quick-wins in there for first-time contributors.
